x-pack/packages/kbn-elastic-assistant/impl/connectorland/connector_selector_inline/action_type_selector_modal.tsx

```diff
@@ -19,13 +19,20 @@ import {
 import { ActionType } from '@kbn/actions-plugin/common';
 import { ActionTypeRegistryContract } from '@kbn/triggers-actions-ui-plugin/public';
 import * as i18n from '../translations';
+import {css} from "@emotion/css/dist/emotion-css.cjs";
 
 interface Props {
   actionTypes?: ActionType[];
   actionTypeRegistry: ActionTypeRegistryContract;
   onClose: () => void;
   onSelect: (actionType: ActionType) => void;
 }
+const itemClassName = css`
```

Comment:
without the extra CSS:
<img width="494" alt="Screenshot 2023-10-16 at 1 52 28 PM" src="https://github.com/elastic/kibana/assets/6935300/fbaa7720-017d-4a76-b9b0-fed4f608ba10">

`white-space` property puts it on one line and aligns the icons
<img width="769" alt="Screenshot 2023-10-16 at 1 53 46 PM" src="https://github.com/elastic/kibana/assets/6935300/d17a2cad-dab2-4c8a-b51a-8b47d136b991">